README.md

```diff
@@ -40,6 +40,47 @@ devtools::install('.')
 BiocBook::preview(BiocBook::BiocBook('.'))
 ```
 
+### Docker installation
+
+Pre-built [Docker images](https://github.com/microbiome/OMA/pkgs/container/oma) are avaialble for an easy deployment.
+
+1. Install Docker for your platform: [Linux](https://docs.docker.com/engine/install/),
+[Mac](https://docs.docker.com/desktop/setup/install/mac-install/), [Windows](https://docs.docker.com/desktop/setup/install/windows-install/).
+2. Pull the image from our repository:
+
+```sh
+docker pull ghcr.io/microbiome/oma:latest
+```
+
+You can also specify any of the available tags to pull a particular version of the image.
+
+3. Switch to the directory of the book, for example, with `cd` command.
+
+4. Run the image:
+
+```sh
+docker run --volume ./:/project -p 8787:8787 -e PASSWORD=1234 ghcr.io/microbiome/oma
+```
+
+The command above will start the image based on the Biocoductor docker image, which includes RStudio Server.
+The `--volume` maps the path on your host (the machine you are using) to the path inside the container.
+In the command, it maps the current working directory to the `/project` directory inside the container.
+If the current directory is the book's repository, it will be available inside the container under `/project`.
+Thus, in this case, the version of the book coincides with the bleeding-edge version available in the repo.
+If you do not specify this option, the book Quarto files are still available in the `/opt/pkg` directory
+inside the container, but any changes you make to the files, will be _discarded_ when you stop the container.
```

Comment:
Add line breaks when the line is over 80 characters long (it seems that some might be).